### PR TITLE
CDE-14 Repository selection step

### PR DESCRIPTION
**Pull request by Aiga115**
Issue #CDE-14
Problem:  Repository selection step
Solution: Created UI for repository selection step
image

### CODE REVIEW

**Review by vidhyalongani — CHANGES_REQUESTED** [review]
@Aiga115 For the tabs in the modal, i have already added them in my other PR. Feel free to update from there. Or you can wait for it to get reviewed and merged

File: lib/components/steps/StepTwo.tsx [hunk 1]
```diff
@@ -1,22 +1,79 @@
...
+    };
+
     return (
         <ModalLayout>
-            <Stack alignItems="center" justifyContent="center" width="100%" sx={{mx: 6}}>
-                <Stack spacing={6} sx={{width: 'max-content'}}>
-                    <Typography> This is step 2</Typography>
+            <>
+                <StyledStepper />
+                <Stack alignItems="center" justifyContent="center" sx={{mx: 3, my: 6}}>
```
Comment (vidhyalongani) [inline lib/components/steps/StepTwo.tsx]: @Aiga115 this will be stepone not two, suggestions would be step one

**Review by vidhyalongani — CHANGES_REQUESTED** [review]
@Aiga115 things looks good. Just few fixes are required

- this whole box should be clickable not just the text

- also the height in figms for the box is 52

Screen Shot 2024-01-08 at 12 14 32 PM

- No padding is required in this one
Screen Shot 2024-01-08 at 12 17 11 PM

File: lib/components/steps/StepOne.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                            >
+                                Select repository
+                            </Button>
+                        </Box>
+                        <Box sx={{ mt: 1.5 }}>
+                            <Button
+                                disableRipple
+                                variant="text"
+                            >
+                                Can’t find the repository you’re looking for? Contact us{' '}
```
Comment (vidhyalongani) [inline lib/components/steps/StepOne.tsx]: @Aiga115 this text should be some link inside the typography

**Comment by zsinnema** [comment]
@Aiga115 there are some merge conflicts, can you please solve them? Thanks!

**Review by vidhyalongani — APPROVED** [review]
@Aiga115 looks good.
@zsinnema this can be merged

**Comment by zsinnema** [comment]
thanks @vidhya-metacell

**Pull request merged**